code/modules/clothing/gloves/boxing.dm

```diff
@@ -17,3 +17,14 @@
 /obj/item/clothing/gloves/boxing/yellow
 	icon_state = "boxingyellow"
 	item_state = "boxingyellow"
+
+/obj/item/clothing/gloves/boxing/yellow/insulated
+	name = "budget boxing gloves"
+	desc = "Standard boxing gloves coated in a makeshift insulating coat. This can't possibly go wrong at all."
+	icon_state = "boxingyellow"
+	item_state = "boxingyellow"
+	siemens_coefficient = 1	//Set to a default of 1, gets overridden in Initialize()
+
+/obj/item/clothing/gloves/boxing/yellow/insulated/Initialize()
+	. = ..()
+	siemens_coefficient = pick(0,0,0,0,0.25,2)
```

Comment:
```suggestion
	if(prob(25)
		siemens_coefficient = rand(0.5, 2) 
	siemens_coefficient = 0
```

Reply:
this change doesn't make sense
it will set it to 0 everytime

Reply:
Never ever use rand() with non-integer values

Reply:
> this change doesn't make sense
> it will set it to 0 everytime

> Never ever use rand() with non-integer values

my bad

Reply:
Your if statement is malformed.